pipeline/tools/amass.yaml

```diff
@@ -1,7 +1,8 @@
 installed: false
 dependencies: [go]
-go: &gotool !get_tool_path "{go}"
-amass: &amass !get_tool_path "{amass}"
+tools: &tools !get_default "{tools-dir}"
+go: &gotool !get_tool_path "{go[path]}"
+path: &amass !join_path [*tools, "amass"]
 
 commands:
 - !join [*gotool, get -u github.com/OWASP/Amass/v3/...]
```

Comment:
amass v3.6.0 introduced gremlin support and the module is throwing an error when the `-u` flag is used during amass install on kali.  recommend removal 

`go get github.com/OWASP/Amass/v3/...`